fundamentals/installation.md

```diff
@@ -2,106 +2,118 @@
 
 Welcome to the **napari** installation tutorial!
 
-This tutorial will teach you how to do a clean install of **napari**. It is aimed at people that just want to use napari. For people also interested in contributing to napari please check our [contributing guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md) for more advanced installation procedures. At the end of the tutorial you should have napari successfully installed on your computer and be able to make the napari viewer appear.
+This tutorial will teach you how to do a clean install of **napari**. It is
+aimed at people that just want to use napari. For people also interested in
+contributing to napari please check our [contributing
+guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
+for more advanced installation procedures. At the end of the tutorial you should
+have napari successfully installed on your computer and be able to make the
+napari viewer appear.
 
 ## installation
 
-**napari** can be installed on most macOS, Linux, and Windows systems with Python 3.6 or 3.7. There are three different ways to install napari.
+**napari** can be installed on most macOS, Linux, and Windows systems with
+Python 3.6 or 3.7. There are three different ways to install napari.
 
 ### install the latest release from pip
- The simplest option is to install the latest release on PyPi by calling
+
+ The simplest option is to install the latest "batteries included" release from
+ PyPi by calling
 
 ```sh
-$ pip install napari
+pip install napari[all]
 ```
 
 ### install from the master branch on Github
+
```

Comment:
Do you want to add the language about virtual environments here too from the main readme?